packages/apps/storybook/package.json

```diff
@@ -14,6 +14,7 @@
     "@itwin/delete-itwin-react": "^2.0.0",
     "@itwin/imodel-browser-react": "~2.3.2",
     "@itwin/itwinui-react": "^2.12.18",
+    "@itwin/itwinui-react_v3": "npm:@itwin/itwinui-react@^3.17.3",
```

Comment:
If an alias is being used, then this won't behave correctly because the package is still expecting to import `@itwin/itwinui-react`. I believe `overrides` will be necessary to get around this.